src/state.sh

```diff
@@ -11,31 +11,31 @@ function State::getTestsPassed() {
 }
 
 function State::addTestsPassed() {
-  ((_TESTS_PASSED++))
+  ((_TESTS_PASSED++)) || true
```

Comment:
This is because when we are running the script with `set -e` to stop on the failing command, the arithmetic operation is evaluated and it's returning a non zero value. 

https://stackoverflow.com/a/66824557

You can run this example:

```
#!/bin/bash

set -e

test_var=0
(( test_var++)) 
echo "$test_var"
```

The return value on `(( test_var++ ))` is 1 so with the `set -e` enabled it will stop in this point. If you add `(( test_var++)) || echo $?` you can see the returned value